routes/user/user.js

```diff
@@ -189,7 +189,8 @@ exports.activateUser = function(req, res, next) {
                 //removing the token and password from the response (security)
                 docs.us_activation_token = undefined;
                 docs.us_password = undefined;
-                res.send(docs);
+                console.log(docs);
+                res.redirect("/#/signin");
```

Comment:
@nick22891 No the redirect must not happen using the backend.  Instead use the front-end application to accept the token as an URL parameter within the app controller.